packages/backend/src/open_payments/payment/outgoing/service.ts

```diff
@@ -89,7 +89,7 @@ async function getOutgoingPayment(
 ): Promise<OutgoingPayment | undefined> {
   const outgoingPayment = await OutgoingPayment.query(deps.knex)
     .get(options)
-    .withGraphFetched('[quote.asset, walletAddress]')
+    .withGraphFetched('[quote.[asset, ilpQuoteDetails], walletAddress]')
```

Comment:
That was not the intention. Why would we? 

This was added when factoring out ilpQuoteDetails to maintain current functionality of the previously implicitly ilp quotes. When there is no `ilpQuoteDetails` this will be `null` which seems fine to me.

I guess maybe this relates back to https://github.com/interledger/rafiki/pull/2857#discussion_r1722025824?